rebasehelper/application.py

```diff
@@ -678,11 +676,29 @@ def get_rebased_patches(self):
             output_patch_string.append('Patches were not touched. All were applied properly')
         return output_patch_string
 
-    def print_summary(self):
-        output = output_tool.OutputTool(self.conf.outputtool)
-        report_file = os.path.join(self.results_dir, self.conf.outputtool + settings.REBASE_HELPER_OUTPUT_SUFFIX)
-        output.print_information(path=report_file)
-        logger.info('\nReport file from rebase-helper is available here: %s', report_file)
+    def print_summary(self, message=None):
```

Comment:
Why is the argument called `message` when it's in fact `Exception` object? Wouldn't `exception` or `error` be more suitable?